### PR TITLE
create animals model

### PR DESCRIPTION
**Pull request by kosniowskiunosquare**
Create animals model with the breed description

### CODE REVIEW

File: src/animals/animals.resolver.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  @Mutation(() => Animal)
+  createAnimal(
+    @Args('createAnimalInput') createAnimalInput: CreateAnimalInput,
+  ) {
+    return this.animalsService.create(createAnimalInput);
+  }
+
+  @Query(() => [Animal], { name: 'AllAnimals' })
+  findAll() {
+    return this.animalsService.findAll();
```
Comment (matthewUnowquare) [inline src/animals/animals.resolver.ts:19]: method findAll does not exist on animal Service, please create it

File: src/animals/animals.resolver.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  }
+
+  @Query(() => Animal, { name: 'breed' })
```
Comment (matthewUnowquare) [inline src/animals/animals.resolver.ts]: change from 
`name: 'breed'`
to
`name: animal`

**Review by matthewUnowquare — CHANGES_REQUESTED** [review]

File: src/animals/animals.resolver.ts (hunk 1 above)
Comment (kosniowskiunosquare) [reply src/animals/animals.resolver.ts:19]: fixed

File: src/animals/animals.resolver.ts (hunk 2 above)
Comment (kosniowskiunosquare) [reply src/animals/animals.resolver.ts]: Fixed

**Review by matthewUnowquare — APPROVED** [review]
LGTM

**Pull request merged**